finished-application/backend/mutations/addToCart.ts

```diff
@@ -1,49 +1,48 @@
-/* eslint-disable */
-import { KeystoneContext, SessionStore } from '@keystone-next/types';
-import { CartItem } from '../schemas/CartItem';
-import { Session } from '../types';
-
-import { CartItemCreateInput } from '../.keystone/schema-types';
+import { KeystoneContext } from "@keystone-next/keystone/types";
+import { KeystoneContext as ProjectKeystoneContext } from ".keystone/types";
+import { CartItem } from ".prisma/client";
+import { Session } from "../types";
 
 async function addToCart(
   root: any,
   { productId }: { productId: string },
-  context: KeystoneContext
-): Promise<CartItemCreateInput> {
-  console.log('ADDING TO CART!');
+  _context: KeystoneContext
+): Promise<CartItem> {
+  const context = _context as ProjectKeystoneContext;
+  console.log("ADDING TO CART!");
   // 1. Query the current user see if they are signed in
   const sesh = context.session as Session;
   if (!sesh.itemId) {
-    throw new Error('You must be logged in to do this!');
+    throw new Error("You must be logged in to do this!");
   }
   // 2. Query the current users cart
-  const allCartItems = await context.lists.CartItem.findMany({
-    where: { user: { id: sesh.itemId }, product: { id: productId } },
-    resolveFields: 'id,quantity'
+  const allCartItems = await context.db.CartItem.findMany({
+    where: {
+      user: { id: { equals: sesh.itemId } },
+      product: { id: { equals: productId } },
+    },
   });
 
   const [existingCartItem] = allCartItems;
   if (existingCartItem) {
-    console.log(existingCartItem)
+    console.log(existingCartItem);
     console.log(
       `There are already ${existingCartItem.quantity}, increment by 1!`
     );
     // 3. See if the current item is in their cart
     // 4. if itis, increment by 1
-    return await context.lists.CartItem.updateOne({
-      id: existingCartItem.id,
+    return context.db.CartItem.updateOne({
```

Comment:
is this change from `lists` to `db` related to the upgrade, or related to sqlite?  if I were using postgres would I still change this to `db`

Reply:
Yes the change is related to the Keystone upgrade, if you were using `PostgreSQL` then it would still be `db` as it refers to the Prisma instance 🙂